docs/data-access/data-download.md

```diff
@@ -50,8 +51,14 @@ docker run --rm -it -e "METADATA_URL=https://song.argo.cancercollaboratory.org"
 
 ### Score-Client with Configured Values
 
-Download the **[latest version of the score-client](https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/score-client/[RELEASE]/score-client-[RELEASE]-dist.tar.gz)**.
-Once you have unzipped the tarball, update the `/conf/application.properties` file with your user values, including:
+Download the **[latest version of the score-client](https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/score-client/[RELEASE]/score-client-[RELEASE]-dist.tar.gz)**. Once you have unzipped the tarball, change directories into the unzipped folder:
```

Comment:
should this say "change directories to the unzipped folder"?

Reply:
i think into is ok ?
![image](https://user-images.githubusercontent.com/15824252/83436117-540daf80-a40b-11ea-89f8-22f80ea041f4.png)